src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int shortBase;
+    private int longBase;
+    private int height;
+
+    public IsoscelesTrapezoid(int shortBase, int longBase, int height) {
+        this.shortBase = shortBase;
+        this.longBase = longBase;
+        this.height = height;
+    }
+
+    @Override
+    public Figure createRandomFigure() {
+        return new IsoscelesTrapezoid(getRandomInt(), getRandomInt(), getRandomInt());
+    }
+
+    @Override
+    public void printInformation() {
+        System.out.println("IsoscelesTrapezoid short base: " + shortBase
+                + " long base: " + longBase + " height: " + height
+                + " area " + (((shortBase + longBase) / 2) * height)
+                + " Color " + getColor());
```

Comment:
All magic numbers in your code should be constants. The formula for area calculation is using magic numbers. Define them as constants with descriptive names.